_artifacts/document-attributes-dm.adoc

```diff
@@ -5,8 +5,8 @@
 :PRODUCT_INIT_BA: rhba
 :PRODUCT_INIT_CAP_BA: RHBA
 
-:PLANNER: Red Hat Business Optimizer
-:PLANNER_SHORT: Business Optimizer
+:PLANNER: Red Hat build of OptaPlanner
```

Comment:
[naming] Some of the changes use {PLANNER} rather than {PLANNER_SHORT}, which means some sentences are presented as "Process Automation Manager, Decision Manager, or Red Hat build of OptaPlanner". Is this correct or should it be "Process Automation Manager, Decision Manager, or OptaPlanner"?

Reply:
The full name is Red Hat build of OptaPlanner. The shortened version is just OptaPlanner, @csherrar .

Reply:
Sorry, I meant to ask if the placement of the full and short name is correct given that some sentences use the short name for other products while the full name is used for OptaPlanner.

Reply:
@csherrar There might be some inconsistency, but the convention I follow is to use the full name of first use in each module, followed by the short name, for OptaPlanner anyway, because as Anna points out Red Hat build of Optaplanner is a mouth full!